tutorial/README.md

```diff
@@ -4,6 +4,8 @@ This guide demonstrates how to generate a pyramidal OME-TIFF [with Bio-Formats](
 ### Getting Started
 
 > NOTE: If you wish to view an image located on a remote machine accessible via SSH, please see the note at the end of this document.
+> NOTE 2: This tutorial does not work on Safari.  We had to make a choice (for now and to the best of our knowledge)
+> between supporting localhost connections in Safari and supporting them in Chrome, and decided to go with Chrome. 
```

Comment:
I would move this note under "Viewing in Avivator". The tutorial works fine, its just viewing the generated data with a local server via `http-server`. Folks can still drag and drop the image in Avivator.

The issue isn't `localhost` connections; it's HTTP v. HTTPS. Browsers block all connections to HTTP from an HTTPS host. Chrome and Firefox allow HTTP connections from HTTPS in the special case for `localhost`, whereas Safari blocks any HTTP connection. If you were to start a local HTTPS server on `localhost` it would work in Safari.

```suggestion

> ⚠️ Warning ⚠️ This section only works in Chrome, Firefox, and Edge (not Safari) due to differences in how browser restrict websites hosted at `https://` URLs (Avivator) from issuing requests to `http://` (the local data server) as a security measure. The supported browsers allow requests to `http://` from `https://` under the special case of `localhost`, whereas Safari prevents all requests to `http://`. As a workaround, you can start an Avivator client at `http://`, but we suggest trying a different supported browser. Alternatively, you can drag-and-drop an image (no local server) into the viewer in any browser.
````